odoo/service/server.py

```diff
@@ -1265,9 +1265,8 @@ def preload_registries(dbnames):
                 _logger.info("Starting post tests")
                 tests_before = registry._assertion_report.testsRun
                 with odoo.api.Environment.manage():
-                    for module_name in module_names:
-                        result = loader.run_suite(loader.make_suite(module_name, 'post_install'), module_name)
-                        registry._assertion_report.update(result)
+                    result = loader.run_suite(loader.make_suite(module_names, 'post_install'), 'base')
```

Comment:
I think that `False` instead of `'base'` would feel less hacky. 
You can also modify this parameter to make it optionnal in run_suite, even cleaner.

Reply:
done